traptor/traptor.py

```diff
@@ -138,6 +139,11 @@ def _setup_birdy(self):
                                         self.apikeys['ACCESS_TOKEN_SECRET']
                                         )
 
+    @retry(wait=wait_exponential(multiplier=1, max=10),
+           stop=stop_after_attempt(3),
+           reraise=True,
+           retry=retry_if_exception_type(KafkaUnavailableError)
```

Comment:
Agreed. I'll keep the one for now and perhaps we can turn your `TenaciousProducer` into a library and use that. This is pretty preliminary.